src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
 }
 ```
 
+## Arterial Spin Labeling perfusion data
+
+Template:
+
+```Text
+sub-<label>/[ses-<label>/]
+    perf/
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_asl.nii[.gz]
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_asl.json
+       sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_aslcontext.tsv
+       [sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_m0scan.nii[.gz]]
+       [sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_dir-<label>][_run-<index>]_m0scan.json]
+       [sub-<label>[_ses-<label>][_acq-<label>][_rec-<label>][_run-<index>]_labeling.jpg]
+```
+
+The complete ASL timeseries should be stored as a 4D NIfTI file in the original acquisition order, accompanied by two ancillary 
```

Comment:
@patsycle my spelling control (Grammarly) also tells me we should use `time series` or `time-series` instead of `timeseries`. Let's choose the one that is most correct (in US spelling - if I'm correct this is the English spelling that BIDS uses?) and use this one throughout.

Reply:
done